### ISSUE TITLE
[Feature request] Upcoming speed limit warning

### ISSUE THREAD

**Issue by ValeriySh** [osmandapp/OsmAnd-iOS#1078]
Similar to "Attention: stop sign" warning it could be useful to warn about upcoming speed limits on your way. 

For example, this is particularly helpful when entering school/playground zones. Since there is no separate definition in the OSM for school zones, they could be detected by looking for the `conditional=<speed> @ <time>` tag.

This feature would also eliminate the time gap between the moment you exceed the current speed limit and the corresponding warning since you have been warned in advance.

**Comment by vshcherb** [comment]
Agree, there is a current speed limit widget and the warning should appear once you go upper that limit

**Comment by ValeriySh** [comment]
Yes, there is a sign in the corner, but also there is a time gap between exceeding the limit and the corresponding warning. I tried to measure the gap driving 1-2 km/h over the limit, sometimes I had driven 100 m before I got a warning. 

I would also suggest a much more visible warning for speed/stops, not just a sign in a corner, but rather a fullscreen flashing one.

Maybe it should be a separate request.
